sandboxmgt/app/forms.py

```diff
@@ -11,6 +13,17 @@ def validate_gov_email(value):
             params={'value': value},
         )
 
+def validate_github(value):
```

Comment:
I'm not sure if should be up to us to validate GitHub accounts. I mean: we can detect invalid chars but we still cannot know if a given account is valid or not. What we can do instead is require the user to login to GitHub **before** requesting the Sandbox: in this way it's impossible to make a mistake. What do you think about?

Reply:
Yes we can do all these steps at some point. This simply implements the first, which is a useful guard against people making silly mistakes.